src/Tools/BuildActionTelemetryTable/BuildActionTelemetryTable.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0-windows</TargetFrameworks>
```

Comment:
Because [this](https://github.com/dotnet/roslyn/blob/f6900176d7d351057077805330281947611d1b4a/src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj#L43) reference does not target `netstandard` or `net6.0` I needed to change the tfm to be windows only for now